sklearn/cross_validation.py

```diff
@@ -300,9 +308,10 @@ class KFold(_BaseKFold):
 
     See also
     --------
-    StratifiedKFold: take label information into account to avoid building
-    folds with imbalanced class distributions (for binary or multiclass
-    classification tasks).
+    StratifiedKFold
```

Comment:
This renders perfectly! Seems like this is the way to go for multiline `<explanation>`s... :)